### PR TITLE
Open Liberty labs

### LINKED ISSUE #3: Add a Jaeger lab using OpenLiberty Microprofile
We have spring boot covered, but we are lacking a lab that shows how to instrument a microprofile java app with opentracing and jaeger

@nastacio is giving a talk and will be contributing this lab 🙌 
https://twitter.com/dnastacio/status/1233151298679328768?s=20

### PR DESCRIPTION
**Pull request by nastacio**
- Standard Docker-based lab
- Istio-based lab

Close #3

### CODE REVIEW

**Comment by nastacio** [comment]
cc @csantanapr , fyi.

**Comment by csantanapr** [comment]
I just added to my TODOs to review it and give it a try thanks for submitting the PR

File: modules/ROOT/pages/ocp-istio-ol.adoc [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+    <artifactId>jaeger-client</artifactId>
+    <version>0.34.0</version>
+</dependency>
+----
+
+. Deploy the Istio Gateway and VirtualService
++
+[source, bash]
+----
+cd lab-jaeger-istio-ol
```
Comment (csantanapr) [inline modules/ROOT/pages/ocp-istio-ol.adoc]:
```suggestion
```
No need for this, they will get an error since they are already in the right directory

**Comment by csantanapr** [comment]
Only found minor problem which we can log an issue or just document a warning

If the user like likes to git clone using ssh git@ instead of https://
Then there is an error in OCP build config istio lab, it assumes the current repo is https and current branch and tries to do a git clone

The. It will required configuration of a secret to allow git clone using ash key

**Comment by nastacio** [comment]
> Only found minor problem which we can log an issue or just document a warning
> 
> If the user like likes to git clone using ssh git@ instead of https://
> Then there is an error in OCP build config istio lab, it assumes the current repo is https and current branch and tries to do a git clone
> 
> The. It will required configuration of a secret to allow git clone using ash key

I think documentation will be best. I need to rebuild the environment to recreate what you found and have the telltale error messages in the warning.

**Pull request merged**